File: 002/002.js
var limit = 4000000;
var total = 0;

let e = 1;
let f = 1;
let i = 1;

while (f < limit) {
  console.log(i, f);
  tmp = e;
  e = f;
  f = f + tmp;
  i++;


  if (f % 2 == 0) {total += f;}
}

console.log(total)

File: 002/002.py
def sum_even_fibs(limit):
    e = 0
    f = 1
    total = 0
    while f < limit:
        e, f = f, e + f

        if f % 2 == 0:
            total += f
   
    return total
    
limit = 4000000
ans = sum_even_fibs(limit)
print(ans)
